### DISCUSSION TITLE
OpenCore Gigabyte Z370n Wifi

### DISCUSSION THREAD

**Discussion by jimpalompa** [jimpalompa/OpenCore-Gigabyte-Z370n-Wifi#4] [General]
Questions and comments goes here, cheers!

**Comment by Nemezus** [comment]
Hi! Which version of BIOS did you use?

**Reply by Nemezus** [reply]
> 

Sorry, saw later F14B. Any plan to upgrade to F14?

**Reply by jimpalompa** [reply]
Hi!

My bad, I was already using F14. Readme is now updated.

**Comment by Boos4721** [comment]
```bash
$ pmset -g log | grep -i fail
2022-08-29 01:28:28 +0800 Failure             	Failure during sleep: 0x0000001F : EFI/Bootrom Failure after last point of entry to sleep
2022-08-29 01:35:06 +0800 Failure             	Failure during sleep: 0x0000001F : EFI/Bootrom Failure after last point of entry to sleep panic
```
any idea?

**Reply by jimpalompa** [reply]
Sorry, no idea. Every build is individual : (

**Reply by Celerii** [reply]
This might help?

https://www.tonymacx86.com/threads/asus-z690-proart-creator-wifi-thunderbolt-4-i7-12700k-amd-rx-6800-xt.318311/post-2336466

**Reply by Boos4721** [reply]
> Sorry, no idea. Every build is individual : (

ok thk

**Reply by Boos4721** [reply]
> This might help?
> 
> https://www.tonymacx86.com/threads/asus-z690-proart-creator-wifi-thunderbolt-4-i7-12700k-amd-rx-6800-xt.318311/post-2336466

i'ill try later.

**Comment by Celerii** [comment]
Excellent share indeed!
I have been experiencing problems with a very similar build (with a i5-8600) since Big Sur and I ended up screwing things up even more while trying to update to Monterey recently.
My config seemed beyond repair, so I started from scratch with your EFI folder. I only modified the Plaform Info and USB mapping to match my tastes, and boom, everything works again.
Thanks a lot!

**Reply by jimpalompa** [reply]
That's awesome to hear! I myself began just like you. I've always wanted to give back to the community, so if people actually can make use of this efi, it's wonderful news 😃

**Comment by cyiooi** [comment]
Hi！I found that replacing SmallTreeIntel82576 with [AppleIGB ](https://github.com/donatengit/AppleIGB)can fix the problem that the i211AT ethernet port is unavailable.
Ventura (13.0) has been tested.
[AppleIGB.kext.zip](https://github.com/jimpalompa/OpenCore-Gigabyte-Z370n-Wifi/files/9861324/AppleIGB.kext.zip)

**Reply by Boos4721** [reply]
ya, https://github.com/donatengit/AppleIGB/releases

**Reply by jimpalompa** [reply]
Good to know, I'll look into it when they publish a stable release. Thanks guys!

**Comment by biandizou** [comment]
Good work!
when  you generate SMBIOS information, choose imac19.1 or imac19.2?

**Reply by biandizou** [reply]
oh, i find it, 19.1

**Comment by biandizou** [comment]
hi,  i7-8700k+z370n+AMD Radeon RX 580 Series ( 4 GB)  +sn570 m2 , my problem is that: after loin in mac os 13.4 with password, the computer will restart,  This question is driving me crazy and I can't find the reason

**Reply by biandizou** [reply]
Memory 16 GB DDR4 2133MHz 8GB x2)

**Reply by biandizou** [reply]
Need to unlock CFG?

**Reply by jimpalompa** [reply]
It's very hard to say, since every system is individual. Sorry, but I can't tell why : (
